dids/src/test/kotlin/web5/sdk/dids/did/BearerDidTest.kt

```diff
@@ -1,20 +1,59 @@
 package web5.sdk.dids.did
 
 import org.junit.jupiter.api.Assertions.assertArrayEquals
+import org.junit.jupiter.api.Assertions.assertTrue
 import org.mockito.kotlin.any
 import org.mockito.kotlin.doReturn
 import org.mockito.kotlin.eq
 import org.mockito.kotlin.spy
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.whenever
 import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.dids.didcore.Service
 import web5.sdk.dids.methods.jwk.DidJwk
 import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertNotNull
 
 class BearerDidTest {
 
+  @Test
+  fun `addService should add a new service to the DID Document`() {
+    val bearerDid = DidJwk.create(InMemoryKeyManager())
```

Comment:
nit: could we do this using DidDht.create, since that's what #310 used to describe the bug?

Reply:
done